gatherers/manifest.js

```diff
@@ -71,7 +72,11 @@ class Manifest extends Gather {
           })
           .then(manifestURL => Manifest._loadFromURL(options, manifestURL));
         })
-        .then(manifest => ({manifest}));
+        .then(manifestContent => {
+          return {
+            manifest: manifestParser(manifestContent).value
```

Comment:
hmm, you may want to pass in just `manifestParser(manifestContent)` and make the auditors do the annoying `.value` dance. For manifest exists, at least, an undefined value means no manifest unless `warning` is set, in which case the JSON was invalid